### PR TITLE
feat(fullstack): new test for fullstack

### PR DESCRIPTION
**Pull request by ruffac**
Adding a new test for senior full stack engineers.

Wanted for the candidate to use a bigger dataset so we can see how he would handle a big amount of data and render that to the user.

Narrowed down to the languages/frameworks/dbs that we are using in order to see fit. Will give them at most 2 weeks if they need to learn the language too (assesses learning aptitude) (and are busy on work).

### CODE REVIEW

**Comment by meetbryce** [comment]
Nice little take home exercise. Lots of freedom here!

Some quick ideas:
- specifying mySQL vs "sql"
- suggesting an example source of data like quandl.com
- indicating how much time should be spent (this might happen outside the test spec though)
- could we tell them we'll assess their git strategy?
- should we tell them how they should submit their finished work e.g. repo or .zip or a live link #golive (could make it bonus points for putting it live on the internet)
- could we have the example make use of Luna elements? or have it more of a wireframe (rather than a mockup)?

**Comment by ruffac** [comment]
Thanks Bryce and DJ! Freedom was an objective!
- Opening the db requirement to any sql database, I think there'll be not much difference on setup and connections. But probably, this is me coming from wanting to have had a chance to be more flexible on the db - including nosql, but our current infra dictates relational. But I'd be happy to update to mySQl if there's anything in particular that we want to assess in there.
- Added quandl.com on the options for data sets, thanks!
- What in particular will we be assessing in git strategy? For a single repo, single dev - it'll be a stretch to veer from trunk workflow. Has adhering to committing principles been a steep climb for some of the new crew members? If yes, we can add this in probably.
- Great idea on #golive! Thank you!
- Thanks for the Luna tip! Linked the Luna guidelines.

**Comment by ademagic** [comment]
Is this PR still relevant?

File: full-stack/dataset-app/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Page Layout + Data Sets
+
+This assessment is for us to see your tech competence fit and for you to have a glimpse of what kind of engineering problems we are solving.
+
+Use any big data public dataset and build the back and front end services to render this info to the user. We’re looking at a UI similar in template as below. Use React for the FE, PHP/Java/nodeJS for the backend and SQL for the db. Apply authentication, add unit and integration tests too. 
+
+Successful candidates will deliver a solution that; applies a clean design pattern, human-centered interaction design and a code base that is evolvable and maintainable.
```
Comment (ademagic) [inline full-stack/dataset-app/README.md]: Would it be helpful to mention how _specifically_ we'd like them to deliver the solution? I.e. a github repo? Just to mention the obvious

File: full-stack/dataset-app/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Page Layout + Data Sets
+
+This assessment is for us to see your tech competence fit and for you to have a glimpse of what kind of engineering problems we are solving.
+
+Use any big data public dataset and build the back and front end services to render this info to the user. We’re looking at a UI similar in template as below. Use React for the FE, PHP/Java/nodeJS for the backend and SQL for the db. Apply authentication, add unit and integration tests too.
```
Comment (ademagic) [inline full-stack/dataset-app/README.md]: > PHP/Java/nodeJS
It might be obvious to the applicant but I'd love to reword this to say "use the language that the role you're applying for requires", but I'm not sure how to word it nicely. E.g. DOPS roles need PHP pros, but someone who has been working in Node recently may apply and decide to use Node for their solution, which wouldn't be ideal.

Comment (ademagic) [inline full-stack/dataset-app/README.md]:
```suggestion
Use any big data public dataset and build the back and front end services to render this info to the user. We’re looking at a UI similar to the template below. Use React for the FE, PHP/Java/nodeJS for the backend and SQL for the db. Apply authentication, add unit and integration tests too. 
```

**Comment by ademagic** [comment]
Do we want to give people a time limit to this? Or a guideline of how long we expect this to take?
